opencsp/common/lib/deflectometry/ParamsSlopeSolverParaboloid.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class ParamsSlopeSolverParaboloid:
```

Comment:
Should this inherit from ParamsSlopeSolver?

Reply:
Wow, I just left these classes unfinished...🤦‍♂️ ParamsSlopeSolver is now ParamsSlopeSolverAbstract and is inherited by ParamsSlopeSolverParabolic and ParamsSlopeSolverPlano. Thanks for the catch!

Reply:
Code reviews ftw